site/app/models/User.php

```diff
@@ -108,6 +108,8 @@ class User extends AbstractModel {
     protected $group;
     /** @prop @var int The access level of the user (ex: superuser, faculty, user) */
     protected $access_level;
+    /** @prop @var bool Should the user only have one active session at a time? */
+    protected $enforce_secure_session;
```

Comment:
Is this the best name for this functionality?  When reading through the rest of this PR, I wasn't sure what this variable did.  Perhaps something like `$enforce_single_session` would be more readable.

Reply:
As suggested, it has been changed to `$enforce_single_session`.